Resources/Prototypes/Entities/Objects/Consumable/Food/produce.yml

```
@@ -1581,6 +1581,61 @@
     tags:
     - Fruit # It's in the name
 
+- type: entity
+  name: capfruit
+  parent: FoodProduceBase
+  id: FoodRealCapfruit
+  description: A soft but smooth gun-shaped fruit.
+  components:
+  - type: FlavorProfile
+    flavors:
+      - capfruit
+  - type: SolutionContainerManager
+    solutions:
+      food:
+        maxVol: 10
+        reagents:
+        - ReagentId: Nutriment
+          Quantity: 5
```

```suggestion
        - ReagentId: Nutriment
          Quantity: 5
        - ReagentId: Sulfur
          Quantity: 5
```
Currently the fake one has sulfur but the real one does not. Shouldn't it be the other way around? And you can distinguish them by flavor due to that, so I would put sulfur in both so you actually have to fire it at something to find out which is real.

---

fake cap gun aka fake fake gun aka gun
real cap gun is just a toy

---

hm, yeah i guess it'll be more fun if both are sulfur. I'll do that.